pipelines/HSC/DRP-ci_hsc.yaml

```diff
@@ -74,6 +74,18 @@ tasks:
       atools.wholeSkyMetric.keysWithBand: parameters.objectTableCoreKeysWithBand
       python: |
         from lsst.analysis.tools.atools import *
+  analyzeMatchedVisitCore:
+    class: lsst.analysis.tools.tasks.AssociatedSourcesTractAnalysisTask
+    config:
+      applyProperMotions: false
```

Comment:
Why disable PM correction in HSC? Wouldn't that still improve any metrics that compare with a refcat?

Reply:
This is just for ci_hsc, where there's not enough data to run `gbdesAstrometricFit`. I added comments here and for the other tasks with this config set.